Dev

EternalVelard opened the pull request:
Все исправил

EternalVelard committed:
modified:   README.md

EternalVelard committed:
modified:   README.md

EternalVelard committed:
new file:   about.html
	new file:   image/main_photo.jpg
	new file:   image/pinterest.png
	new file:   image/smb1.jpg
	new file:   image/smb2.jpg
	new file:   image/smb3.jpg
	new file:   image/smb4.jpg
	new file:   image/telegram.png
	new file:   image/twitter.png
	new file:   image/vk.png
	new file:   image/youtube.png
	new file:   index.html
	new file:   script.js
	new file:   styles.css
	new file:   video/train.mp4

EternalVelard committed:
Merge branch 'dev' of https://github.com/stepancar-web-programming/2023-fall-lab-portfolio-chechetko-v.-v. into dev

EternalVelard committed:
Merge pull request #2 from stepancar-web-programming/dev

	modified:   README.md

EternalVelard committed:
Merge pull request #4 from stepancar-web-programming/dev2

Dev2

EternalVelard committed:
modified:   index.html

EternalVelard committed:
Merge branch 'dev' of https://github.com/stepancar-web-programming/2023-fall-lab-portfolio-chechetko-v.-v. into dev

EternalVelard committed:
modified:   about.html

EternalVelard committed:
modified:   about.html
	new file:   external-script.js
	new file:   footer.html
	new file:   header.html
	modified:   index.html

EternalVelard committed:
modified:   README.md
	modified:   footer.html

stepancar reviewed (changes requested):
Не хватает автоматизации проверок. В проекте нет eslint/stylelint. editorconfig

EternalVelard committed:
new file:   .eslintrc.js
	new file:   .stylelintrc
	modified:   about.html
	modified:   external-script.js
	new file:   footer.css
	new file:   header.css
	modified:   header.html
	modified:   index.html
	modified:   script.js
	modified:   styles.css

EternalVelard committed:
new file:   .editorconfig

EternalVelard commented:
> Не хватает автоматизации проверок. В проекте нет eslint/stylelint. editorconfig

Все сделано

EternalVelard committed:
modified:   README.md

stepancar reviewed (changes requested):
В проекте появились конфиги для eslint/stylelint, но как они запускаются, устанавливаются не ясно. На CI нет запуска никаких проверок.

EternalVelard committed:
Create action.yml

EternalVelard committed:
modified:   .editorconfig
	deleted:    .eslintrc.js
	new file:   .eslintrc.json
	new file:   .gitignore
	deleted:    .stylelintrc
	new file:   .stylelintrc.json
	modified:   about.html
	modified:   external-script.js
	new file:   package-lock.json
	new file:   package.json
	modified:   script.js
	modified:   styles.css

EternalVelard committed:
modified:   README.md

EternalVelard commented:
> В проекте появились конфиги для eslint/stylelint, но как они запускаются, устанавливаются не ясно. На CI нет запуска никаких проверок.

Сделал

stepancar reviewed (changes requested):
В пайплайне есть warnings
<img width="1478" alt="Screenshot 2024-03-21 at 07 35 33" src="https://github.com/stepancar-web-programming/2023-fall-lab-portfolio-chechetko-v.-v./assets/2854910/685cf577-14c0-4dd1-8ea3-62f3ca5ed7e4">

EternalVelard committed:
Update action.yml

EternalVelard commented:
> В пайплайне есть warnings <img alt="Screenshot 2024-03-21 at 07 35 33" width="1478" src="https://private-user-images.githubusercontent.com/2854910/315332830-685cf577-14c0-4dd1-8ea3-62f3ca5ed7e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEwMzc3MTMsIm5iZiI6MTcxMTAzNzQxMywicGF0aCI6Ii8yODU0OTEwLzMxNTMzMjgzMC02ODVjZjU3Ny0xNGMwLTRkZDEtOGVhMy02MmYzY2E1ZWQ3ZTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjFUMTYxMDEzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTMxM2JkY2VlMGM5MjgzOGQwZmY1YmZhNGM0NjhlMTZlZmU4MThiYThiMmZiNjU0ZWM5ZDI2MWJkMDk2MDU2OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.l3i6quVPEdk1EI4xOKTB3iL9Fw1e_QGQWELodSbD3ZU">

исправил

stepancar commented on about.html:
```diff
@@ -0,0 +1,84 @@
+
+<body>
+    <header id="header-placeholder"></header>
+    <style>
+        .flex {
+            display: flex;
+        }
+
+        .flex-container {
+            flex: 50%;
+            display: flex;
+            align-items: center;
+        }
+    </style>
```
Неоднотипность подходов; Какие-то стили вынесены в css файл, какие-то определены тут

EternalVelard replied:
Исправил. Теперь все стили в css файлах

stepancar commented on about.html:
```diff
@@ -0,0 +1,84 @@
+                    <div class="carousel right-space">
+                        <div class="slider">
+                            <div class="slide">
+                                <img src="image/smb1.jpg" alt="1" />
```
пожалуйста, убедитесь в том, что назначение аттрибута alt вам понятно

EternalVelard replied:
Как понимаю alt содержит текстовое описание изображения. И если бы браузер у нас не читал их или в проекте его не было, вместо изображения и была это надпись. Alt не обязателен тут

EternalVelard replied:
По сути он даёт понять при отсутствие или отключении изображения в браузере, что это за изображение. В мое случае либо убрать, потому что 1,2,3 не очень информативно, либо как я и сделал более информативно и поменял названия

EternalVelard replied:
Так же думаю можно это использовать для проверки работы слайдера, если бы изображения были отключены

stepancar commented on footer.css:
```diff
@@ -0,0 +1,51 @@
+    margin-right: 0.25rem;
+  }
+  
+  .icons a{
```
Кажется вы не валидируете форматирование

EternalVelard replied:
Отформатировал все файлы

EternalVelard committed:
modified:   about.html
	modified:   external-script.js
	modified:   footer.css
	modified:   header.css
	modified:   styles.css

EternalVelard committed:
Merge branch 'dev' of https://github.com/stepancar-web-programming/2023-fall-lab-portfolio-chechetko-v.-v. into dev

EternalVelard committed:
modified:   about.html

stepancar commented on about.html:
```diff
@@ -0,0 +1,73 @@
+                    <div class="carousel right-space">
+                        <div class="slider">
+                            <div class="slide">
+                                <img src="image/smb1.jpg" alt="smb_train_photo1" />
```
Alt аттрибут должен содержать описание изображения на картинке. Слепой пользователь не сможет увидеть картинку на вашем сайте, но он сможет всеравно понять что перед ним если вы добавите описание. Скрин ридер прочитает текст. 
Тут вы можете увидеть как примерно слепые пользуются сайтами.
https://www.youtube.com/watch?v=RQiN1Hhrxu0
alt это всего лишь один из тегов чтобы обеспечить доступность web. Внесенные вами изменения не позволят понять что за контент smb_train_photo1 - не читаемая строка. возможно, поисковый робот с гибкой языковой моделью сможет разобраться, но обычные скрин ридеры - нет

EternalVelard replied:
Исправил сделал более информативно
![smb_photo1](https://github.com/stepancar-web-programming/2023-fall-lab-portfolio-chechetko-v.-v./assets/87513856/5835332d-4add-48d5-be85-2e4312f16791)

EternalVelard committed:
modified:   about.html
